src/crypto/tls/handshake_server_tls13.go

```diff
@@ -922,6 +928,8 @@ func (hs *serverHandshakeStateTLS13) readClientCertificate() error {
 		}
 
 		hs.transcript.Write(certVerify.marshal())
+
+		hs.handshakeTimings.ReadCertificateVerify = hs.handshakeTimings.elapsedTime()
```

Comment:
I think this should be run unconditionally. (Right now it's only run if there is a client certificate.) Maybe move this  below on line 934?

Reply:
That's a good idea, yeah. Stand by.